### PR TITLE
add books-module containing (create, display,update and delete) views

### PR DESCRIPTION
**Pull request by Brianktoo**
add `create, display, update and delete features` to manipulate books dictionary list

### CODE REVIEW

File: library-management-system/.gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+*/__pycache__/
+.env
```
Comment (codetaskmaster) [inline library-management-system/.gitignore:2]: Whitespace!

File: library-management-system/app/__init__.py [hunk 2]
```diff
@@ -0,0 +1,5 @@
+from flask import Flask
+
+def create_app():
+    app = Flask(__name__)
+    return app
```
Comment (codetaskmaster) [inline library-management-system/app/__init__.py:5]: Whitespace!

**Review by codetaskmaster — CHANGES_REQUESTED** [review]

File: library-management-system/.gitignore (hunk 1 above)
Comment (Brianktoo) [reply library-management-system/.gitignore]: Done

**Review by codetaskmaster — APPROVED** [review]

**Comment by codetaskmaster** [comment]
LGTM! Proceed

**Review by codetaskmaster — COMMENTED** [review]
Great job @Brianktoo 

LGTM. Proceed!

**Pull request merged**